verify/vm-prep

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
```

Comment:
Because stuff in cockpit changes between pull requests. #6334 removes these from the cockpit repo. This cockpit1 network is legacy and in the future will only be used for testing non-master branches in the cockpit repo.